sync/src/main/java/org/bf2/sync/ManagedKafkaSync.java

```diff
@@ -103,6 +103,25 @@ public void syncKafkaClusters() {
             String localKey = Cache.namespaceKeyFunc(remoteManagedKafka.getMetadata().getNamespace(), remoteManagedKafka.getMetadata().getName());
             ManagedKafka existing = lookup.getLocalManagedKafka(localKey);
 
+            // special handling if the control plane indicates paused
+            if (remoteManagedKafka.getAnnotation(ManagedKafkaKeys.Annotations.PAUSE_RECONCILIATION).map(Boolean::valueOf).orElse(false)) {
+                if (!remoteManagedKafka.getSpec().isDeleted()) {
+                    log.debugf("Taking no action on ManagedKafka %s as the control plane has paused it", Cache.metaNamespaceKeyFunc(remoteManagedKafka));
+                    continue;
+                }
+                if (existing != null) {
+                    if (!existing.getSpec().isDeleted()) {
+                        log.debugf("Passing just the delete flag to the ManagedKafka %s", Cache.metaNamespaceKeyFunc(remoteManagedKafka));
+                        kubeClient.resources(ManagedKafka.class)
+                                .inNamespace(existing.getMetadata().getNamespace())
+                                .withName(existing.getMetadata().getName())
+                                .patch(PatchContext.of(PatchType.JSON),
+                                        "[{\"op\": \"replace\", \"path\":\"/spec/deleted\", \"value\":true}]");
```

Comment:
Why this style instead of using the POJO?

Reply:
Strategic merge is not supported.  There isn't a "good" direct way of producing the simplified json patch.  And json merge isn't great for partial documents, but may be fine here, I didn't check specifically.